src/puppetlabs/puppetdb/command.clj

```diff
@@ -428,20 +414,27 @@
         (recur (dec n))
```

Comment:
I assume the Throwable catch above is what pdb already does, but while reviewing this I wondered about it, i.e. do we really want to keep trying after a virtual machine InternalError, etc.